packages/buffetjs-core/src/components/Toggle/index.js

```diff
@@ -10,7 +10,7 @@ import PropTypes from 'prop-types';
 import { Toggle as StyledToggle, ToggleWrapper } from '@buffetjs/styles';
 import Label from '../Label';
 
-function Toggle({ disabled, id, className, name, onChange, value }) {
+function Toggle({ disabled, id, className, name, onChange, value, labelLeft, labelRight }) {
```

Comment:
Hello @raphael-ms,

Can you rename the props to `rightLabel` and `leftLabel` please?

Reply:
yes, sorry for that. done.

Reply:
What about `falseLabel` & `trueLabel` or `checkedLabel` & `uncheckedLabel`? Aren't they better suited?